mcs/packages/Makefile

```diff
@@ -3,7 +3,7 @@ include ../build/rules.make
 
 ROSLYN_CSC_DIR = $(dir $(CSC_LOCATION))
 
-ROSLYN_FILES = \
+ROSLYN_FILES_FOR_MONO = \
```

Comment:
This is to copy `ROSLYN_FILES_FOR_MSBUILD` to `lib/mono/msbuild/15.0/bin/Roslyn`. This does not copy `csc.exe`.